### PR TITLE
docs(wrapper): сlarify attachedToDocument meaning in docs

### LINKED ISSUE #1207: Clarify attachedToDocument meaning in docs
### Version
1.0.0-beta.29

### Reproduction link
[https://vue-test-utils.vuejs.org/api/wrapper/#properties](https://vue-test-utils.vuejs.org/api/wrapper/#properties)

### Steps to reproduce
visit https://vue-test-utils.vuejs.org/api/wrapper/#properties

### What is expected?
Clear description of options

### What is actually happening?
See vague description of options

---
Can't find any meaningful description of what does `attachedToDocument` option for.

### PR DESCRIPTION
**Pull request by dsseng**
Fixes #1207

### CODE REVIEW

File: docs/api/wrapper/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ A `Wrapper` is an object that contains a mounted component or vnode and methods
 
 #### `options.attachedToDocument`
 
-`Boolean` (read-only): True if `attachToDocument` in mounting options was `true`
+`Boolean` (read-only): True if component is attached to DOM when rendered.
```
Comment (eddyerburgh) [inline docs/api/wrapper/README.md]:
```suggestion
`Boolean` (read-only): True if component is attached to document when rendered.
```

**Comment by eddyerburgh** [comment]
@Alex-Sokolov could you review the Russian translation please?

**Review by Alex-Sokolov — APPROVED** [review]

**Comment by Alex-Sokolov** [comment]
@eddyerburgh looks good!

**Review by eddyerburgh — APPROVED** [review]
Thanks :)

**Pull request merged**